From 034445d45f5620f4f3b9f5b5e5b85002f7753e22 Mon Sep 17 00:00:00 2001
From: dmeier <dmeier@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Fri, 6 Mar 2020 15:19:34 +0000
Subject: [PATCH 366/462] MANTIS 56312: Extend Desko-Penta Device-Handler by
 Accufill-SDK for reading US driving-licenses

Introducing environment variable SYS_GDIPLUS

git-svn-id: svn://localhost/SelfServiceBuild/trunk@672 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.common.mak | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SelfServiceBuild/Build/cpp/makefile.common.mak b/SelfServiceBuild/Build/cpp/makefile.common.mak
index b611122..5e07470 100644
--- a/SelfServiceBuild/Build/cpp/makefile.common.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.common.mak
@@ -408,6 +408,7 @@ LIB_MATINIT      = $(_LIB)\matinit.lib
 SYS_ADVAPI32     = advapi32.lib
 SYS_COMCTL32     = comctl32.lib
 SYS_GDI32        = gdi32.lib
+SYS_GDIPLUS      = gdiplus.lib
 SYS_IMAGEHLP     = imagehlp.lib
 SYS_MPR          = mpr.lib
 SYS_SHELL32      = shell32.lib
-- 
2.41.0.windows.1

